docs/en/02_Features/blog_recipe.md

```diff
@@ -70,63 +70,64 @@ For configuration of anti-spam please see the [Akismet configuration guide](/how
 For the basic configuration, see the default `mysite/_config/blog.yml` below for reference:
 
 
-	:::yml
-	---
-	Name: mywidgetsconfig
-	Only:
-	  moduleexists: widgets
-	---
-	# Disable if you do not use widgets on your site
-	SiteTree:
-	  extensions:
-	    - WidgetPageExtension
-	---
-	Name: mycommentsextension
-	Only:
-	  moduleexists: comments
-	---
-	# Enable page comments on the site by default, including frontend moderation / approval
-	SiteTree:
-	  extensions:
-	    - CommentsExtension
-	  comments:
-	    enabled: false
-	    frontend_moderation: true
-	    require_moderation_nonmembers: true
-	    require_moderation_cms: true
-	    require_login: false
-	    require_login_cms: true
-	    nested_comments: true
-	    order_comments_by: '"Created" ASC'
-	---
-	Name: myblogconfig
-	Only:
-	  moduleexists: blog
-	---
-	# Customise the email notification template here
-	BlogPost:
-	  default_notification_template: 'BlogCommentEmail'
-	  comments:
-	    enabled: true
-	---
-	Name: akismetconfig
-	Only:
-	  moduleexists: akismet
-	---
-	# Customise your akismet configuration here
-	SiteConfig:
-	  extensions:
-	    - AkismetConfig
-	# Allows spam posts to be saved for review if necessary
-	AkismetSpamProtector:
-	  save_spam: true
-	---
-	Name: mycommentspamprotection
-	Only:
-	  moduleexists: spamprotection
-	  classexists: CommentingController
-	---
-	# Enable spam protection for comments by default
-	CommentingController:
-	  extensions:
-	    - CommentSpamProtection
+```yml
+---
+Name: mywidgetsconfig
+Only:
+  moduleexists: widgets
```

Comment:
Copy the current version of https://github.com/silverstripe/recipe-blog/blob/master/mysite/_config/blog.yml - this needs namespaces and has had a couple of changes recently